config.go

```diff
@@ -91,6 +91,9 @@ func getConfig() *types.Configuration {
 	v.SetDefault("AWS.CloudWatchLogs.LogGroup", "")
 	v.SetDefault("AWS.CloudWatchLogs.LogStream", "")
 	v.SetDefault("AWS.CloudWatchLogs.MinimumPriority", "")
+	v.SetDefault("AWS.S3.MinimumPriority", "")
+	v.SetDefault("AWS.S3.Bucket", "")
+	v.SetDefault("AWS.S3.Prefix", "falcosidekick")
```

Comment:
```suggestion
	v.SetDefault("AWS.S3.Bucket", "")
	v.SetDefault("AWS.S3.Prefix", "falco")
	v.SetDefault("AWS.S3.MinimumPriority", "")
```

I prefer to keep same order than others and it's more logical to use `falco` as default prefix.